client/src/pages/Login/index.tsx

```diff
@@ -3,8 +3,39 @@ import backgroundImage from "@/assets/images/register-background.svg";
 import FilledButton from "@/components/FilledButton";
 import qrCode from "@/assets/images/qr-code.png";
 import { Link } from "react-router-dom";
+import { toast } from "react-toastify";
+import { AuthContext } from "@/contexts/auth";
+import { useContext } from "react";
 
 const Login = () => {
+  const { setUser } = useContext(AuthContext);
+
+  const handleLogin = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    const formData = new FormData(event.currentTarget);
+
+    const data = Object.fromEntries(formData.entries());
+
+    fetch(`${import.meta.env.VITE_API_ROOT_URL}/login`, {
+      method: "POST",
```

Comment:
I suggest using `axios` and create and axiosInstance to provide the shared settings between requests for example: `baseurl` and other config, what do you think ?